GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -758,6 +767,26 @@ protected void invokeUrl(String serverBase, String urlString,
         return errorMap;
     }
 
+    /**
+     * When a blank or null cell is incurred while processing, make sure that this is the last cell
+     * in the row and ignore any other "phantom cells" that may occur. We only allow this for the
+     * header row. If the blank cell occurs in between valid header cells we return true.
+     *
+     * @param cell
+     * @return
+     */
+    private boolean isMissingHeaderCell(Cell cell) {
+        assert cell.getRow().getRowNum() == 0; // only process header rows
```

Comment:
The code will run under FLOW services so no worries about GAE support.  In this case the main motivation is to ensure the method is only used for checking cells in the header row.